### ISSUE TITLE
Dynalog Bank B Leaf Order 

### ISSUE THREAD

**Issue by bvangenderen** [jrkerns/pylinac#160]
Possible discrepancy between the Log Analyzer documentation and the implementation for indexing leaves when loading Dynalog files. When reading the documentation:
![image](https://user-images.githubusercontent.com/45474475/52825083-3549a500-3070-11e9-8e28-23ab0111914a.png)

I was expecting to get the following leaf pairs (for a standard 120 leaf MLC):
![image](https://user-images.githubusercontent.com/45474475/52825286-0122b400-3071-11e9-8f18-6d44d00cb784.png)

But the ordering I seem to be getting is like this:
![image](https://user-images.githubusercontent.com/45474475/52825336-2b747180-3071-11e9-9716-7e3e157c10f6.png)

I think this is the responsible code:
![image](https://user-images.githubusercontent.com/45474475/52825541-f4529000-3071-11e9-8215-053f7322e6ea.png)

**Comment by jrkerns** [comment]
Why were you expecting the first ordering?

**Comment by mchamberland** [comment]
The documentation explicitly says that leaves 1 and 120, 2 and 119, and 60 and 61 pairs, which corresponds to the first ordering. Also, the documentation says (A_leaf, B_leaf) = (n, N_leaves +1 - 1), which agrees with the first ordering...

**Comment by jrkerns** [comment]
Ah, I must be dyslexic. Got the first and second ordering reversed. Maybe I should read twice first >.>

**Comment by jrkerns** [comment]
Issues should be fixed, looking at one more thing to double check. Resulting fluence should be the same as before if I've not missed anything. Thanks!

**Comment by jrkerns** [comment]
So after further investigation, I realized that both Dynalogs and Trajectory logs worked the way you pointed out @bvangenderen, where documentation said pairs were 1/120, 2/119, ... . After I changed the behavior of dynalogs to match the behavior listed in the documentation I only now realized that the trajectory log behavior was affected. Rather than change both dynalogs and trajectory logs to match the doc behavior, which would be more work for both classes, I will revert the code change and change the documentation to be of the 1/61, 2/62 type. I will be adding more rigorous tests to the fluence calculations to detect these changes. Until now I was spot checking one value for ease. 

TL;DR It's easier to alter the documentation to the current/past behavior than vic versa.
